doc/changelog/08-tools/11255-master+fix11254-coqtop-version.rst

```diff
@@ -0,0 +1,4 @@
+- **Fixed:**
+  `coqtop --version` was broken when called in the middle of an installation process
+  (`#11255 <https://github.com/coq/coq/pull/11255>`_, by Hugo Herbelin, fixing
+   `#11254 <https://github.com/coq/coq/pull/11254>`_).
```

Comment:
It seems indentation here broke the refman build

Reply:
I mixed up the syntax. It is supposed to be ``` ``coqtop --version`` ```. Fixed.

Reply:
Indentation was wrong also. Also fixed.